docs/source/posts/2022/2022-08-17-week-09-shivam.rst

```diff
@@ -0,0 +1,31 @@
+Week 9 - First working skeletal animation prototype
+===================================================
+.. post:: August 16 2022
+   :author: Shivam Anand
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+- This week I had the first working example of skeletal animation ready. I was able to render the ``SimpleSkin`` model. Here's a quick preview:
+
+    .. raw:: html
+
+        <iframe id="player" type="text/html"   width="600" height="390" src="https://user-images.githubusercontent.com/74976752/184981715-a83a8a36-f2bf-4ff1-9366-0c302dcf3f9b.mp4" frameborder="0"></iframe>
+
+- I wrote a custom interpolator (just like the tangent cubic spline interpolator) for interpolating two transformation matrices.
+
+
+What is coming up next week?
+----------------------------
+
+- Generalising the skinning code to work with other models as well.
+- Creating a custom interpolator to interpolate more than 4 matrices at once.
```

Comment:
From what I understand, you need maximum of four matrices for that, but what I'm sure of is that sometimes you need four. so maybe  `interpolate more than 4 matrices at once.` can be `interpolate up to 4 matrices at once.` or `interpolate 4 or more matrices at once.`  depending on what you want.

Reply:
No, We might need to interpolate more than 4  matrices here ( We'll have at least one  transformation matrix for each sampler)